src/encryptionService.ts

```diff
@@ -0,0 +1,97 @@
+import { CopilotSettings } from "@/settings/SettingsPage";
+import { Platform } from "obsidian";
+
+// Dynamically import electron to access safeStorage
+// @ts-ignore
+let safeStorage: Electron.SafeStorage;
+
+if (Platform.isDesktop) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  safeStorage = require("electron")?.remote?.safeStorage;
```

Comment:
This will require a refactor to abstract the safeStorage for other platforms.

We shouldn't double-down on being platform dependant